doc/operator-scope.md

```diff
@@ -46,6 +49,9 @@ With the above changes the specified manifests should look as follows:
     ```YAML
     apiVersion: apps/v1
     kind: Deployment
+    metadata:
+      name: memcached-operator
+      namespace: <operator-namespace>
```

Comment:
The above is not required as well. When we run `kubectl apply -f deploy/operator.yaml` it will be applied in the current namespace and then, we also can  `kubectl apply -f deploy/operator.yaml -n <mynamespace>`.

Reply:
See comment below https://github.com/operator-framework/operator-sdk/pull/2582#discussion_r382906922